src/AccessibilityInsights.Extensions.AzureDevOps/AzureDevOpsIntegration.cs

```diff
@@ -491,27 +491,43 @@ internal Uri CreateIssuePreview(string projectName, string teamName, IReadOnlyDi
         }
 
         /// <summary>
-        /// Retains characters within ascii range below 127 and returns
-        /// an escaped representation of the string
+        /// Substitutions for characters when creating the bug URL
+        /// </summary>
+        internal static IReadOnlyDictionary<char, char> CharacterSubstitutions = new Dictionary<char, char>()
+        {
+            { '�', '-' }, // middle dot (183) to dash (45)
+        };
+
+        /// <summary>
+        /// replaces characters if substitution defined and
+        /// returns an escaped representation of the string
         /// </summary>
         /// <param name="str"></param>
         /// <returns></returns>
         internal static string EscapeForUrl(string str)
         {
+            // characters such as the middle dot (ascii 183) can 
```

Comment:
>characters such as the middle dot (ascii 183) [](start = 15, length = 45)

I don't think I'd mention the 183 value here, since it's handled in the table and this code just follows the table

Reply:
updated